src/PrestaShopBundle/Command/SecurityAnnotationLinterCommand.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShopBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Routing\Route;
+
+final class SecurityAnnotationLinterCommand extends ContainerAwareCommand
+{
+    public function configure()
+    {
+        $this
+            ->setName('prestashop:linter:security-annotation')
+            ->setDescription('Checks if Back Office route controllers has configured Security annotations.')
```

Comment:
```suggestion
            ->setDescription('Checks if Back Office route controllers have mandatory Security annotations.')
```

Reply:
Are you sure it's mandatory and there are no exceptions for it?

Reply:
Indeed 👏